sqldelight-studio-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -52,6 +52,8 @@
   </module-components>
 
   <extensions defaultExtensionNs="com.intellij">
+    <compileServer.plugin classpath="sqldelight-studio-plugin-0.4.3-SNAPSHOT.jar"/>
```

Comment:
right this needs to be updated with each release. I'll add a note to the releasing readme (unless you know how to do it automatically)